system/otherLoadTest/playlist.xml

```diff
@@ -294,14 +292,13 @@
 		</groups>
 		<platformRequirements>^os.win</platformRequirements>
 		<subsets>
-			<subset>14+</subset>
+			<subset>11+</subset>
 		</subsets>
 	</test>
 	
 	<test>
 		<testCaseName>HeapHogLoadTest</testCaseName>
 		<variations>
-			<variation>NoOptions</variation>
```

Comment:
I do not think we need to modify `HeapHogLoadTest `. If you want to remove `<variation>NoOptions</variation> `, please remove the whole block `<variations>`.

Reply:
Change in `HeapHogLoadTest` was a typo actually. Updated the PR.